src/plugins/intel_npu/src/plugin/npuw/lazy_tensor.cpp

```diff
@@ -4,42 +4,55 @@
 
 #include "lazy_tensor.hpp"
 
-using ov::npuw::weights::ConcatMeta;
-using ov::npuw::weights::ConstPtr;
 using ov::npuw::weights::LazyTensor;
-using ov::npuw::weights::OrigData;
-using ov::npuw::weights::Transform;
-using ov::npuw::weights::TransformType;
-using ov::npuw::weights::UnpackMeta;
 
 namespace ov {
 namespace npuw {
 namespace weights {
 
+enum class TransformType : int { TENSOR, CONST, CONCAT, UNPACK, PERMUTE, CONVERT };
+using ConcatMeta = std::pair<std::vector<LazyTensor>, std::size_t>;
+using UnpackMeta = std::tuple<LazyTensor, LazyTensor, LazyTensor, ov::Shape, ov::element::Type>;
+using ConstPtr = std::shared_ptr<ov::op::v0::Constant>;
+using Transform = std::variant<ov::Tensor, ConstPtr, ConcatMeta, UnpackMeta, std::vector<std::size_t>, std::monostate>;
```

Comment:
what is `std::vector<size_t>` in this list? Nobody knows. Give the poor guy a name. It must be one a type defined like in above.

Also, why do you need `monostate`? What does it mean? It seems (referring to the above) `op::Const` is your starting point in any LT hierarchy. Or it is not a no-OP, but fp32_to_f16 conversion? Give it a name too.

Reply:
Done